pandas-stubs/io/stata.pyi

```diff
@@ -147,65 +125,52 @@ class StataWriter(StataParser):
         self,
```

Comment:
StataWriter is not public, but `StataWriter.write_file()` is, so we can just simplify this and only include the `write_file()` method